src/actions/network.js

```diff
@@ -65,27 +72,24 @@ export function connectToNetwork (endpoint, tesseraEndpoint) {
  * successful and showing an error if unsuccessful.
  *
  * @param endpoint Geth RPC Url (http://localhost:22000)
- * @param tesseraEndpoint Tessera party keys endpoint (http://localhost:9081/partyinfo/keys)
+ * @param tesseraEndpoint Tessera endpoint (http://localhost:9081)
  * @returns thunk middleware dispatch function
  */
 export function saveNetwork (endpoint = '', tesseraEndpoint = '') {
   return async dispatch => {
     try {
-      // TODO clearing tessera endpoint every time since we are hiding that field for now. Delete this line later
-      tesseraEndpoint = ''
 
       if (tesseraEndpoint.endsWith('/')) {
         tesseraEndpoint = tesseraEndpoint.substring(0,
           tesseraEndpoint.length - 1)
       }
 
-      await testUrls(endpoint, tesseraEndpoint)
-
       // helper to automatically find partyinfo endpoint when adding a known local node
       if(endpoint.startsWith('http://localhost:2200')) {
         tesseraEndpoint = await getLocalPartyInfoIfAvailable(endpoint)
       }
 
+      await updateTesseraUrl(endpoint, tesseraEndpoint)
```

Comment:
This section was supposed to just call the testUrls method without actually updating the urls that the plugin is using. Then when we dispatch the connectToNetwork action it will actually update the urls. If we update the tessera urls, the test call fails, and the user cancels editing the endpoints, then we will be in a weird state.